bots/discord/docker-compose.example.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  websocket-api:
+    container_name: revanced-discord-bot
```

Comment:
Should probably rename this to revanced-bot-discord-bot just so a. it works with the package filter when purging containers and b. its more consistent with naming (prefix revanced-bot for all packages). Make this change where necessary in other files and places as well.